packages/ubuntu_desktop_installer/lib/app.dart

```diff
@@ -2,6 +2,7 @@ import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:intl/intl.dart';
 import 'package:provider/provider.dart';
+import 'package:ubuntu_desktop_installer/pages/who_are_you_page.dart';
 import 'package:yaru/yaru.dart' as yaru;
```

Comment:
is there any chance we could keep `yaru.dart` dependencies in `app_theme.dart`?

we can provide custom `ThemeData` extensions like this: https://github.com/canonical/ubuntu-desktop-installer/commit/67f6241d88ef6f1842586ab9a5edc9ce6371e19f#diff-ae7dce7083af6843f072a97280a2bed2bed57742c87086bfe1000ac979121c30R34-R48

Reply:
I could use theme.of(context).... if this helps to get the yaru dep outside of this file?
only problem is the green, since in material elevated buttons are just colored with accent color, so we've overridden those in yaru.dart
but I could use

```dart
  Widget findPassWordStrengthLabel(
      BuildContext context,
      String value,
      String weakPasswordLabel,
      String averagePasswordLabel,
      String strongPasswordLabel) {
    final _strongPattenPasswordValidator =
        PatternValidator(r'(?=.*?[#?!@$%^&*-])', errorText: '');
    final _goodPatternPasswordValidator =
        PatternValidator(r'(^.*(?=.{6,})(?=.*\d).*$)', errorText: '');
    if (_strongPattenPasswordValidator.isValid(value)) {
      return Text(strongPasswordLabel,
          style: TextStyle(
            color: Theme.of(context)
                .elevatedButtonTheme
                .style!
                .backgroundColor!
                .resolve({MaterialState.dragged}),
          ));
    } else if (_goodPatternPasswordValidator.isValid(value)) {
      return Text(averagePasswordLabel,
          style: TextStyle(
            color: Theme.of(context).colorScheme.primary,
          ));
    }
    if (MultiValidator([
      RequiredValidator(errorText: ''),
      MinLengthValidator(2, errorText: ''),
    ]).isValid(value)) {
      return Text(weakPasswordLabel,
          style: TextStyle(
            color: Theme.of(context).colorScheme.error,
          ));
    }

    return Text('');
  }
 ```